src/main/java/com/github/manolo8/darkbot/core/IDarkBotAPI.java

```diff
@@ -118,6 +119,20 @@ default void setVisible(boolean visible, boolean fullyHideEnabled) {
     void refine(long refineUtilAddress, OreAPI.Ore ore, int amount);
     long callMethod(int index, long... arguments);
 
+    // Handler API
+    void clearCache();
+    void emptyWorkingSet();
+    void setLocalProxy(int port);
+    void setPosition(int x, int y);
+    void setFlashOcxPath(String path);
+    void setClientSize(int width, int height);
+    void setMinClientSize(int width, int height);
+    void setTransparency(int transparency);
+    void setVolume(int volume); // 0 - 100
+
+    // LOW = 0, MEDIUM = 1, HIGH = 2, BEST = 3, AUTO_LOW = 4, AUTO_HIGH = 5
+    void setQuality(int quality);
```

Comment:
Should use an enum, then internally resolve as val.ordinal(), makes more sense from the outside